packages/editor-ui/src/components/NodeCredentials.vue

```diff
@@ -439,16 +452,18 @@ export default defineComponent({
 
 			const selectedCredentials = this.credentialsStore.getCredentialById(credentialId);
 			const selectedCredentialsType = this.showAll ? selectedCredentials.type : credentialType;
-			const oldCredentials = this.node.credentials?.[selectedCredentialsType]
+			const oldCredentials: INodeCredentialsDetails | null = this.node.credentials?.[
+				selectedCredentialsType
+			]
 				? this.node.credentials[selectedCredentialsType]
-				: {};
+				: null;
```

Comment:
I think it would be clearer to not use nullish coalescing here with ternary, but instead either

1.
```ts
const oldCredentials = this.node.credentials ? this.node.credentials[selectedCredentialsType] : null
```

OR

2.
```ts
const oldCredentials = this.node.credentials?.[selectedCredentialsType] ?? null
```

Reply:
Agree. Will update.